src/auth/feedback-secret.guard.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Env } from 'src/env';
+
+@Injectable()
+export class FeedbackSecretGuard implements CanActivate {
+  constructor() {}
+
+  canActivate(context: ExecutionContext): boolean {
```

Comment:
PullProofAI

Consider adding input validation to ensure that `context`, `request`, and `request.headers` are not null or undefined before accessing their properties to prevent potential runtime errors.
```diff
+     if (!context || !request || !request.headers) {
+         throw new Error('Invalid context or request object');
+     }
```


<!-- This is an auto-generated comment by PullProofAI -->